COPYRIGHT

```diff
@@ -0,0 +1,7 @@
+threshold_crypto is copyright 2018, POA Network Developers.
+
+Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+http://www.apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+http://opensource.org/licenses/MIT>, at your option. All files in the project
+carrying such notice may not be copied, modified, or distributed except
+according to those terms.
```

Comment:
Can you add a newline here?

Reply:
Sorry, a newline where@

Reply:
Just after the last sencence. Github displays a warning about the missing newline at the end of file.

Reply:
It's not code so don't worry about it :)